src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(a) {
```

Comment:
Please, use more descriptive names
```suggestion
  return function adder(num) {
```